server/beslog2json.py

```diff
@@ -385,7 +387,9 @@ def timing_log_to_json(log_fields, json_log_record):
     """
     debug("Processing TIMING_MESSAGE_TYPE")
     send_it = False
-    if TRANSMIT_TIMING_LOG:
+
+    transmit_log = TRANSMIT_TIMING_LOG or (TRANSMIT_PROFILING_LOG and log_fields[12].startswith("Profile timing:"))
```

Comment:
I was able to manually test the `square_bracket_timing_record()` path, but not this `timing_log_to_json` path; not sure if there are tests somewhere that I don't see to test this behavior? Or is a switch for outputting this alternate log type from the bes, so I can at least test manually?

Reply:
You can test it manually by taking grabbing a timing/profiling  line from the `/var/log/bes/bes.log` where the records are separated by `|&|` . The square bracket business is for log lines (early in the startup) where each value is enclosed in square brackets: `[time stuff][pid][message stuff]`